src/System.Resources.ResourceManager/ref/project.json

```diff
@@ -1,14 +1,11 @@
 {
   "dependencies": {
-    "System.Runtime": "4.0.0",
-    "System.Reflection": "4.0.0",
-    "System.Globalization": "4.0.0"
+    "System.IO.UnmanagedMemoryStream": "4.3.0-beta-24604-02",
```

Comment:
After a chat with @weshaggard we decided to push UMS down to System.Runtime.InteropServices.
It can't go as low as System.Runtime because it depends on SafeBuffer.